internal/handlers/tigris/msg_find.go

```diff
@@ -95,8 +95,16 @@ func (h *Handler) MsgFind(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg, er
 	return &reply, nil
 }
 
-// fetchAndFilterDocs fetches documents from the database and filters them using the provided QueryParam.Filter.
-func fetchAndFilterDocs(ctx context.Context, dbPool *tigrisdb.TigrisDB, qp *tigrisdb.QueryParam) ([]*types.Document, error) {
+// fetchAndFilterDocs fetches documents from the database and filters them using the provided QueryParams.Filter.
+func (h *Handler) fetchAndFilterDocs(ctx context.Context, dbPool *tigrisdb.TigrisDB, qp *tigrisdb.QueryParams) ([]*types.Document, error) { //nolint:lll // argument list is too long
```

Comment:
The problem with converting those functions back to methods (as they were before) is that receiver has a lot of extra information, including more `*tigrisdb.TigrisDB` connection pools. I think we should move to functions, not into the opposite direction

Reply:
@AlekSi so then we pass whatever we need from `Handler` as params? Do we need to introduce a new struct to pass params, or should we just extend the list of params?

Reply:
Up to you

Reply:
fixed